frontend/src/components/Course/Course.tsx

```diff
@@ -13,25 +15,61 @@ import {
   ButtonsContainer,
   CourseDescriptionWrapper,
   CourseInfoBox,
+  MobileCourseInfoBox,
+  MobileCourseProgress,
+  MobileCourseCompleted,
+  CourseTextContainer,
 } from './styled';
 
 interface Props {
   title: string | undefined;
   description: string;
   duration: string | undefined;
   lessons: number | undefined;
+  pageName?: string;
+  status?: string;
+  progress?: number;
 }
 
-const CourseItem: React.FC<Props> = ({ title, description, duration, lessons, children }) => (
+const PAGES = {
+  myCourses: 'myCourses',
+  coursesList: 'coursesList',
+};
+
+const CourseItem: React.FC<Props> = ({
+  title,
+  description,
+  duration,
+  lessons,
+  pageName,
+  status,
+  progress,
+  children,
+}) => (
   <CourseContainer container direction="column">
     <AboutCourseContainer>
       <ImageWrapper>
         <Image />
       </ImageWrapper>
-      <CourseTitle>{title}</CourseTitle>
-      <CourseDescriptionWrapper>
-        <CourseDescription>{shortifyCourseDescription(description)}</CourseDescription>
-      </CourseDescriptionWrapper>
+      <CourseTextContainer>
+        <CourseTitle>{title}</CourseTitle>
+        <CourseDescriptionWrapper>
+          <CourseDescription>{shortifyCourseDescription(description)}</CourseDescription>
+        </CourseDescriptionWrapper>
+        <MobileCourseInfoBox>
+          <CourseInfo duration={duration} lessons={lessons} />
+        </MobileCourseInfoBox>
+        {status === 'completed' && (
```

Comment:
We should reuse as many strings as possible. It's not a good practice to fix in each comparison the same string!
Breaks all the paradigms of the good codebase!

Reply:
Moved it to constants